pages/_app.tsx

```diff
@@ -177,7 +177,7 @@ function MyApp({ Component, pageProps }: MyAppProps) {
               (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
               (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
               m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-              })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+              })(document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
This doesn't seem intentional, right? it's outside the scope of this work and a different feature.

Reply:
hmm yea probably from some global search and replace.

Reply:
This is the only update that needs to be made in #256